src/main/kotlin/com/cognifide/gradle/aem/instance/InstanceUrl.kt

```diff
@@ -15,7 +15,22 @@ class InstanceUrl(raw: String) {
         get() = userPart(1) ?: Instance.PASSWORD_DEFAULT
 
     val httpUrl: String
-        get() = "${config.protocol}://${config.host}:${config.port}"
+        get() = if (config.port != -1) {
+            "${config.protocol}://${config.host}:${config.port}"
+        } else {
+            "${config.protocol}://${config.host}"
+        }
+
+    val httpPort: Int
+        get() = if (config.port != -1) {
+            config.port
+        } else {
+            if (config.protocol == "https") {
+                443
+            } else {
+                80
```

Comment:
needed when `localInstance` is being used, then that value is being injected into var `CQ_HTTP_PORT` inside start/start.bat script